### DISCUSSION TITLE
404 error after deploying

### DISCUSSION THREAD

**Discussion by phongo1** [vercel/community#5549] [Help]
### Summary

Having issues with deploying my React app. I set the framework preset to vite and the commands should be correct. It's deploying successfully but getting a 404 deployment not found error. I've also tried the vercel.json file with the rewrites. Thank you In advance.

### Example

https://phong-portfolio-7gmcu33yj-phongo1.vercel.app/

### Steps to Reproduce

Used Vite and React.

**Comment by leerob** [comment]
Is the code open source?

**Reply by phongo1** [reply]
I have resolved the issue - thank you
